app/src/controllers/organization/projects/sagas.js

```diff
@@ -36,7 +37,7 @@ function* watchFetchProjects() {
 }
 
 function* createProject({ payload: { newProjectName: projectName } }) {
-  const { id: organizationId, slug: organizationSlug } = yield select(activeOrganizationSelector);
+  const { id: organizationId, slug: organizationSlug } = yield select(activeOrganizationIdSelector);
```

Comment:
Will it work?
Looks like you are trying to destructuring the number here.

Reply:
Done.